boards/arm/mimxrt1015_evk/mimxrt1015_evk.dts

```diff
@@ -55,6 +55,7 @@ arduino_serial: &uart4 {};
 		compatible = "adesto,at25sf128a", "jedec,spi-nor";
 		reg = <0>;
 		status = "okay";
+		jedec-id = <0x1f 0x89 0x01>;
```

Comment:
Matches https://www.adestotech.com/wp-content/uploads/AT25SF128A_85C_Data_Sheet_RevD_03-19-2019.pdf
Does not match https://www.jedec.org/document_search?search_api_views_fulltext=jep106 (Adesto manufacturer id 0x43)